static/css/style.css

```diff
@@ -221,7 +221,7 @@ blockquote {
 code {
     background-color: #dee5e9;
     color: #00416a;
-    font-family: 'Fira Code', Fallback, monospace;
+    font-family: 'Fira Code', Helvetica, Arial, monospace;
```

Comment:
This is loading a monospace custom font (Fira Code) but the system fallbacks are sans-serif (Helvetica, Arial). That will look a little weird if (when?) the custom fonts don't load. Consider instead:

```
"Fira Code", "Courier New", Courier, monospace;
```

Reply:
Resolved by 2be2784